helm/charts/cloud-carbon-footprint/README.md

```diff
@@ -0,0 +1,119 @@
+<!--- app-name:  cloud-carbon-footprint -->
+
+# Helm Chart for Cloud Carbon Foot Print
+
+Cloud Carbon Footprint is an application that estimates the energy (kilowatt hours) and carbon emissions (metric tons CO2e) of public cloud provider utilization.
+
+
+## TL;DR
+
+```console
+git clone https://github.com/appnetwise/cloud-carbon-footprint.git
+cd helm/charts/cloud-carbon-footprint
+helm install cloud-carbon-footprint .
+```
+
+## Introduction
+
+[Cloud Carbon Footprint](https://www.cloudcarbonfootprint.org) is an application that estimates the energy (kilowatt hours) and carbon emissions (metric tons CO2e) of public cloud provider utilization.
+
+If you would like to learn more about the various calculations and constants that we use for the emissions estimates, check out the [Methodology page](https://www.cloudcarbonfootprint.org/docs/methodology)
+
+
+
+## Prerequisites
+
+- Kubernetes
+- Helm 
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install my-release .
+```
+
+## Parameters
+### Client Configuration
```

Comment:
This is a hefty and detailed list, thanks for including! I recommend moving it to the bottom of the ReadMe since it can shroud the section for how to use/customize the params which is arguably most important and should be at the top

Reply:
Yeah referred to some opensource helmcharts and gave introduction at the top.